### PR TITLE
[FCE-219] Prepare everything for releasing

### PR DESCRIPTION
**Pull request by karkakol**
## Description

Prepare packages for releasing
New release process is described in RELEASE.md

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -2522,6 +2512,16 @@ __metadata:
   languageName: unknown
   linkType: soft
 
+"@fishjam-dev/js-server-sdk@npm:^0.1.0":
```
Comment (mironiasty) [inline yarn.lock:2515]: nit: you can update `webdriverio-test` to use `@fishjam-cloud/js-server-sdk`

**Review by mironiasty — APPROVED** [review]
There are still few leftovers from `fishjam-dev`, but it could be fixed in another PR.

Other than than that, is seem to be fine.

File: .github/workflows/all_packages_release.yaml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        env:
+          COCOAPODS_TRUNK_TOKEN: ${{ secrets.COCOAPODS_TRUNK_TOKEN }}
+        run: |
+          pod lib lint FishjamCloudClient.podspec
+
+      - name: Run npm publish check
+        run: npm publish --dry-run --verbose --access public --tag ${{ steps.tag_name.outputs.TAG_NAME }}
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+        working-directory: ./packages/android-client
```
Comment (graszka22) [inline .github/workflows/all_packages_release.yaml:46]: we release android-client to npm?

File: RELEASE.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+## Release process:
+
+1. Create new branch
+
+- update s.version to x.y.z in [FishjamCloudClient.podspec](./FishjamCloudClient.podspec)
+- update s.version to x.y.z in [RNFishjamClient.podspec](./packages/react-native-client/ios/RNFishjamClient.podspec)
+- update version to x.y.z in [package.json](./packages/react-native-client/package.json)
+- update version to x.y.z in [package.json](./package.json)
+- update version to x.y.z in [build.gradle](./packages/react-native-client/android/build.gradle)
```
Comment (graszka22) [inline RELEASE.md:9]: I want a script to do this automatically, no way I'm editing manually 5 files

File: examples/video-chat/ios/Podfile [hunk 4]
```diff
@@ -49,8 +49,10 @@ target 'FishjamExample' do
       :mac_catalyst_enabled => false
     )
   end
-  pod 'FishjamClient', :path => '../../../packages/ios-client/'
+#   pod 'FishjamCloudClient'
+  pod 'FishjamCloudClient', :path => '../../../'
```
Comment (graszka22) [inline examples/video-chat/ios/Podfile:53]: why this change, and why leftover comment?

File: .github/workflows/all_packages_release.yaml (hunk 2 above)
Comment (karkakol) [reply .github/workflows/all_packages_release.yaml:46]: fixed

File: examples/video-chat/ios/Podfile (hunk 4 above)
Comment (karkakol) [reply examples/video-chat/ios/Podfile:53]: ios was released before, so new name is necessary.

File: RELEASE.md (hunk 3 above)
Comment (karkakol) [reply RELEASE.md:9]: For now i would leave it as it is. If this way will work as expected, then script can be created. Another thing is that there will be more files to update ( e.g. `read.me`, documentation in future) in almost every package. It would require some fixed/known structure.

File: examples/video-chat/ios/Podfile (hunk 4 above)
Comment (graszka22) [reply examples/video-chat/ios/Podfile:52]: and the path is correct?

**Review by graszka22 — APPROVED** [review]

**Review by mironiasty — APPROVED** [review]
